extensions/swagger-ui/deployment/src/main/java/io/quarkus/swaggerui/deployment/SwaggerUiProcessor.java

```diff
@@ -203,7 +203,7 @@ public String updateApiUrl(String original, String openApiPath) {
     @ConfigRoot
     static final class SwaggerUiConfig {
         /**
-         * The path of the swagger-ui servlet.
+         * The path of the swagger-ui route.
```

Comment:
Not entirely sure about that one. It might be a bit too technical?

Reply:
Well we should either make the proposed change or remove the classification. The current documentation is just wrong 😎.

Also route is pretty generic I think, so it could pass as non technical

Reply:
`endpoint` maybe or still the same?

Reply:
Ah I had not seen @geoand comment.

Reply:
Maybe  `Path where Swagger UI is mounted`?

Reply:
Or available?

Reply:
Path would be best IMHO

Reply:
let's go with `Path where Swagger UI is available` :-) Thanks @geoand @gsmet

Reply:
Thanks for fixing it @machi1990 !